### PR TITLE
Downgraded ipywidget hack

### PR DESCRIPTION
**Pull request by parente**
Fixed whitespace for jon's PR #1278

### CODE REVIEW

**Comment by jdfreder** [comment]
@parente my hero!

**Comment by jdfreder** [comment]
This needs a little bit more work

**Comment by jdfreder** [comment]
@parente can you remove the WIP on this, it's working now.  cc @minrk

File: notebook/notebookapp.py [hunk 1]
```diff
@@ -271,17 +271,21 @@ def init_handlers(self, settings):
...
         widgets = None
         try:
-            import ipywidgets as widgets
-            handlers.append(
-                (r"/nbextensions/widgets/(.*)", FileFindHandler, {
-                    'path': widgets.find_static_assets(),
-                    'no_cache_paths': ['/'], # don't cache anything in nbextensions
-                }),
-            )
+            import widgetsnbextension
```
Comment (parente) [inline notebook/notebookapp.py]: Oh noes!

**Pull request merged**